src/internal/delegates/mtu-delegate.ts

```diff
@@ -0,0 +1,43 @@
+import { SimulatedBleError, BleErrorCode } from "../../ble-error";
+import { SimulatedPeripheral } from "../../simulated-peripheral";
+import { errorIfUnknown, errorIfConnected, errorIfNotConnected, errorIfBluetoothNotOn, errorConnectionFailed } from "../error_creator";
+import { AdapterState, ConnectionState, Subscription } from "../../types";
+import { Platform } from 'react-native';
+
+export class MtuDelegate {
+
+    async requestMtu(
+        adapterState: AdapterState,
+        peripherals: Map<string, SimulatedPeripheral>,
+        peripheralIdentifier: string,
+        mtu: number
+    ): Promise<SimulatedBleError | number | undefined> {
```

Comment:
```suggestion
    ): Promise<SimulatedBleError | number> {
```
Typescript will whine unless you add a return statement in `else` block of catch, so you need to incorporate that suggestion as well.